lxd/resources/cpu.go

```diff
@@ -41,42 +41,51 @@ func GetCPUIsolated() []int64 {
 	return isolatedCpusInt
 }
 
-// ParseCpuset parses a limits.cpu range into a list of CPU ids.
-func ParseCpuset(cpu string) ([]int64, error) {
-	cpus := []int64{}
-	chunks := strings.Split(cpu, ",")
+func parseRangedListToInt64Slice(input string) ([]int64, error) {
+	res := []int64{}
+	chunks := strings.Split(input, ",")
 	for _, chunk := range chunks {
 		if strings.Contains(chunk, "-") {
 			// Range
 			fields := strings.SplitN(chunk, "-", 2)
 			if len(fields) != 2 {
-				return nil, fmt.Errorf("Invalid cpuset value: %s", cpu)
+				return nil, fmt.Errorf("Invalid cpuset value: %s", input)
```

Comment:
You've removed all the other error contexts, but left this one so, it stands out as odd. 
Given my other comment around improving/changing the contexts rather than removing them, this one should like also be changed.